FlowCrypt/Core/Core.swift

```diff
@@ -80,13 +80,11 @@ actor Core: KeyDecrypter, KeyParser, CoreComposeMessageType {
             "armored": armoredPrv
         ]
 
-        let verifyResult = try await call(
+        return try await call(
             "verifyKey",
             jsonDict: jsonDict,
             data: nil
-        )
-
-        return Data()
+        ).data
```

Comment:
What does this endpoint newly return? I suppose originally it was just empty, meaning if verification failed it would be error. Not sure what it's returning now. If still nothing much, we should change signature to `func verifyKey(armoredPrv: String) async throws -> void {`

Reply:
This endpoint is used only for testing of keys verification performance.

`verifyKey` method of `Core` returns empty data if key is verified, which is the same as previous `Data()` (that's why I removed it).
But your proposal with returning `void` looks better 👍